### DISCUSSION TITLE
How to remove leading plus and minus signs?

### DISCUSSION THREAD

**Discussion by jozuas** [magit/magit#4548] [Support] [answered]
![image](https://user-images.githubusercontent.com/13085275/143767240-5c7e7eab-2117-45c3-a8e5-086feacd8dcd.png)

I would like to remove leading `+` and `-` signs as colour communicates the same meaning.

**Comment by tarsius** [comment, marked as answer]
That is not supported and I don't intend to add it because a simple implementation (using overlays that hide the first column) would impact performance considerably and actually removing the first (or more for merges) column would complicate many things and introduce bugs.
